Jenkinsfile

```diff
@@ -101,6 +101,19 @@ pipeline {
               }
             }
           }
+          post {
+            always {
+              steps {
+                sh '''
+                  rm -rf k-distribution/k-tutorial/1_basic/build
+                  rm -rf kframework-${env.VERSION}
+                  make --directory=k-distribution/tests/regression-new clean
+                  make --directory=k-exercises clean
```

Comment:
Does this also clean up stuff in `k-distribution/`?

Reply:
the `make --directory=k-exercises clean` command should clean up `k-distribution/pl-tutorial`. But that's all besides the other two commands that deal with that subdirectory.

I guess `make --directory=k-distribution clean` would cover more of it. @ehildenb Should we also try to clean up `k-distribution/target`? I think that's ~500M.